modules/ting_search_ortograf/ting_search_ortograf.module

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * @file
+ * Add auto-complete functions to the ting search field, using the Ortograf service.
+ */
+
+/**
+ * Implements hook_menu().
+ */
+function ting_search_ortograf_menu() {
+  $items = array();
+
+  $items['ting/autocomplete'] = array(
```

Comment:
Please add `ortograf ` to the path. It is especially important here as another module uses the exact same menu path.